kubernetes/pom.xml

```diff
@@ -66,6 +78,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-release-plugin</artifactId>
+        <version>2.5.3</version>
+        <!--
+          During release:perform, enable the "release" profile
```

Comment:
It's invoked right here: https://github.com/kubernetes-client/java/pull/49/files#diff-0bbb33e5f782eef602668a770a9215caR91